scrapy/utils/display.py

```diff
@@ -3,24 +3,62 @@
 """
 
 import sys
+from platform import version
+from packaging.version import parse
 from pprint import pformat as pformat_
 
 
 def _colorize(text, colorize=True):
     if not colorize or not sys.stdout.isatty():
         return text
     try:
+        import pygments.util
+        from pygments.formatters import get_formatter_by_name
         from pygments import highlight
-        from pygments.formatters import TerminalFormatter
         from pygments.lexers import PythonLexer
-        return highlight(text, PythonLexer(), TerminalFormatter())
     except ImportError:
         return text
 
+    """
+    All Windows versions >= "10.0.14393" interpret ANSI escape sequences
+    using terminal processing.
+
+    Enable enivornment variable `ENABLE_VIRTUAL_TERMINAL_PROCESSING`
+    to activate terminal processing.
+    """
```

Comment:
:lipstick: Docstrings are not meant to be used in the middle of a function implementation.

```suggestion
    # All Windows versions >= "10.0.14393" interpret ANSI escape sequences
    # using terminal processing.
    #
    # Enable enivornment variable `ENABLE_VIRTUAL_TERMINAL_PROCESSING`
    # to activate terminal processing.
```